kits/default/bcl-builder.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require("path");
+
+const outputFolder = path.resolve(__dirname);
+const nodeModules = path.resolve(__dirname, "./node_modules");
+
+// SCSS includePaths
+const includePaths = [nodeModules];
+
+module.exports = {
+  styles: [
+    {
+      entry: path.resolve(outputFolder, "resources/sass/default.style.scss"),
+      dest: path.resolve(outputFolder, "assets/css/oe_bootstrap_theme.style.min.css"),
```

Comment:
replace `oe_bootstrap_theme` with placeholder, add it also to .libraries.yml

Reply:
Updated 👍